Create markdown documentation for scenario templating engine in V11

marcboulle opened the pull request:
- reprise de la doc existante sur les scenarios: KhiopsScenarios.pptx du repo khiops-pdf-docs
- extension avec les nouvelles fonctionnalites V11 de pilotage par un fichier de parametres JSON
- integration dans le plan, en fin de section "How to/Deployment & Integration"

marcboulle committed:
Create markdown documentation for scenario templating engine in V11

- reprise de la doc existante sur les scenarios: KhiopsScenarios.pptx du repo khiops-pdf-docs
- extension avec les nouvelles fonctionnalites V11 de pilotage par un fichier de parametres JSON
- integration dans le plan, en fin de section "How to/Deployment & Integration"

sgouache commented:
Contenu OK et facile à lire, bon travail! 

Quelques coquilles ou remarques stylistiques çà et là:

- -O <file>: same as -o option, but without replay -> je suppose que c'est same as -j

- Begin by opening a Khiops shell, which can be done by selecting the Shell Khiops option from the Khiops menu, accessible via the Start button in Windows.
-> Incompréhensible pour un utilisateur non-windows, quel est l'effet de cette action?

- Examples khiops -r -> les commandes multiligne windows semblent terminées par un ^, il faudrait indiquer que sous linux/macos le caractère ^ sera avantageusement remplacé par un \

- All lines within this block are repeated as many times as necessary, based on the input data. -> input data referenced by "loop key" ?

- Lines within this block are conditionally executed depending on the test result. -> if the key variable is present and equal to true 

- The array key identifies (<loop key>) a list of scenario lines within a loop. -> The array key (<loop key>) identifies a list of scenario lines within a loop.
Note: JSON objects, key/value pair, est-ce qu'il ne s'agit pas de tuples, auquel cas il faudrait le dire "array contains json tuples with a consistent length" et parler ensuite des current tuple's element

- The boolean key identifies (<if key>) a list of scenario lines inside a conditional block. -> The boolean key (<if key>) identifies a list of scenario lines inside a conditional block.

- Après Usage Examples / Input JSON file: 
chemin C:\Users\Public\khiops_data\samples\Iris\Iris.kdic -> je suppose qu'il manque une section Input dictionary avec un bloc de code contenant le dico

- Section "Handling Non-UTF8 values": dans les exemples, préferer la syntaxe JSON:
   - "dataFile": "/tmp/journées.txt"
   - "byteDataFile": "L3RtcC9qb3VybsOpZXMudHh0"

marcboulle committed:
Feedback from Stephane

Toutes les remarques sont prises en compte, sauf les commandes qui continuent à utiliser le caret '^' (et non le bbackslah),
en coherence avec les chemins d'acces aux samples qui sont presentes pour Windows.

carinehue commented:
Relecture rapide : ci-joint quelques coquilles relevées
[scenario.md](https://github.com/user-attachments/files/21040165/scenario.md)

Coquille dans mkdocs.yml : Distributed au lieu de Distibuted

carinehue reviewed (approved):
LGTM

marcboulle committed:
Feedback from Carine

marcboulle committed:
Remove tutotial and scenario PDF from doc V11
